terraform/simple_instance/ec2.tf

```diff
@@ -29,6 +29,7 @@ resource "aws_security_group" "ssh_traffic" {
 resource "aws_instance" "web_server_instance" {
   ami             = data.aws_ami.ubuntu.id
   instance_type   = "t2.micro"
+
```

Comment:
```suggestion
  monitoring = true
}
```
<details>
    <summary>
        <img src="https://bc-network-graph-icons.s3.us-west-2.amazonaws.com/prCommentsSeverityIcons/medium.svg" alt="MEDIUM" width="64px" align="center">
            &nbsp;&nbsp;<b style="color:black" >Detailed monitoring for EC2 instances is disabled</b>
            <br>
               &nbsp;&nbsp;&nbsp;&nbsp;Resource: <a href="https://github.com/bds1959/terragoat/blob/bds-devops/terraform/simple_instance/ec2.tf#L19-L26">aws_instance.web_server_instance</a>  |  ID: <code>BC_AWS_LOGGING_26</code>
            <br></summary>
<h4>Description</h4>
The Instance Metadata Service (IMDS) is an on-instance component used by code on the instance to securely access instance metadata. You can access instance metadata from a running instance using one of the following methods:
 * Instance Metadata Service Version 1 (IMDSv1) – a request/response method
 * Instance Metadata Service Version 2 (IMDSv2) – a session-oriented method

As a request/response method IMDSv1 is prone to local misconfigurations: 
 * Open proxies, open NATs and routers, server-side reflection vulnerabilities.
 * One way or another, local software might access local-only data.

<h4>Benchmarks</h4>
<ul>
<li>FEDRAMP (MODERATE) AC-6</li>
</ul>

</details>